client/app/scripts/superdesk-workspace/content/content.js

```diff
@@ -18,32 +18,53 @@
                 'type'
             ];
 
+            var scope = $scope;
+
             /**
              * Create an item and start editing it
              */
             this.create = function(type) {
+                if (scope && scope.dirty){
```

Comment:
It's better to move this conditional in closeOpenNew() then to repeat it in multiple methods.